tests/spread/integration/apt/task.yaml

```
@@ -0,0 +1,25 @@
+summary: Integration tests for apt
+
+execute: |
+  rootfs="$(install-slices apt_get-bins)"
+
+  mkdir -p "${rootfs}/dev"
+  mount --bind /dev "${rootfs}/dev"
+  cp /etc/resolv.conf "${rootfs}/etc/resolv.conf"
+
+  chroot "${rootfs}/" apt update
+
+  chroot "${rootfs}/" apt install python3.12 -y
+
+  chroot "${rootfs}/" python3.12 -c 'print("Hello, World!")' | grep -q "Hello, World!"
+
+  umount "${rootfs}/dev"
+
+  rootfs="$(install-slices apt_placeholder)"
+
+  mkdir -p "${rootfs}/dev"
+  mount --bind /dev "${rootfs}/dev"
+
+  chroot "${rootfs}/" apt | grep -q "chisel cut --root / apt_"
+
+  umount "${rootfs}/dev"
```

```suggestion

  chroot "${rootfs}/" apt | grep -q "chisel cut --root / apt_"

```
Not sure we need `/dev` for this task.